Makefile

```diff
@@ -4,7 +4,7 @@
 PREFIX=/usr/local
 BINDIR=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man
-ALL_CFLAGS=$(CFLAGS) -std=c99 -Wall -Wextra -Wshadow -Wpedantic
+ALL_CFLAGS=-std=c99 -Wall -Wextra -Wshadow -Wmissing-prototypes -Wpedantic
```

Comment:
Why remove $(CFLAGS) from here? `ALL_CFLAGS` is intended to contain all flags needed to compile the source files.

Reply:
I moved it down to the actual rule so that the line would be shorter, but I moved it back now.